src/effects/EffectUI.cpp

```diff
@@ -1139,8 +1139,8 @@ void EffectUIHost::LoadUserPresets()
 void EffectUIHost::InitializeRealtime()
 {
    if (mSupportsRealtime && !mInitialized) {
-      mpState = RealtimeEffectManager::Get(*mProject)
-         .AddState(nullptr, PluginManager::GetID(&mEffect));
+      mpState =
```

Comment:
test: open project with one track, instantiate a BassTreble before ever pressing play.

Again mpState will be returned nullptr, but this time because of a different reason:

the call to GetOwningProject() inside AudioIO::AddState will return nullptr.
this happens because the owning project is not set until `AudioIO::StartPortAudioStream` is reached, but that happens only after pressing play.

Reply:
Please watch out: even if I press play (and then stop) to set the owning project, and only then I instantiate a BassTreble, `AudioIO::AddState` will still return nullptr, but this time because `mpTransportState `will be null.

Reply:
> Please watch out: even if I press play (and then stop) to set the owning project, and only then I instantiate a BassTreble, `AudioIO::AddState` will still return nullptr, but this time because `mpTransportState `will be null.

Let's try this once more.  I made changes to fix this.